jsk_robot_common/jsk_robot_startup/scripts/shutdown.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import os
+import rospy
+from std_msgs.msg import Empty
+
+
+class Shutdown(object):
+    """
+    This node shuts down or reboots the robot itself
+    according to the rostopic.
+
+    Note that this node needs to be run with sudo privileges.
+
+    Usage:
+    # Launch node
+    $ su [sudo user] -c ". [setup.bash]; rosrun jsk_robot_startup shutdown.py"
+
+    # To shutdown robot
+    rostopic pub /shutdown std_msgs/Empty
+    # To restart robot
+    rostopic pub /reboot std_msgs/Empty
+    """
+
+    def __init__(self):
+        rospy.loginfo('Start shutdown node.')
+        rospy.Subscriber('shutdown', Empty, self.shutdown)
+        rospy.Subscriber('reboot', Empty, self.reboot)
+
+    def shutdown(self, msg):
+        rospy.loginfo('Shut down robot.')
+        shutdown_command = '/sbin/shutdown -h now'
```

Comment:
can you change this node to customize shutdown command from `rosparam`?
for pr2, we need to use `pr2-shutdown`

Reply:
Thank you for your review!
That makes sense.

I updated.
[`661b4f6` (#1426)](https://github.com/jsk-ros-pkg/jsk_robot/pull/1426/commits/661b4f6a42dd32b39e0b4e3841c229be93b53f48)